src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,39 +2,44 @@
 
 public class DefaultCountingOutRhymer {
 
-    private static final int SIZE = 12;
+    private static final int ARRAY_SIZE = 12;
 
-    private static final int FULL = 11;
+    private static final int CAPACITY = 11;
```

Comment:
This constant is related to SIZE. Changing only one leads to an error (inconsistency). The name is wrong (FULL_RHYMER_INDICATOR)